### DISCUSSION TITLE
How to enable ethernet on RJ45 featured cards

### DISCUSSION THREAD

**Discussion by bashers222** [raomin/ESPAltherma#276] [General]
I'm having issues with wifi signal with ESP8266 cards fitted in the Daikin control box
I've since bought an ESP32-ETH and can enable the ethernet drivers so it can be cabled
However i cant get the config to ignore wifi and attempt to connect via ethernet.
Any hints?

**Comment by raomin** [comment]
What is not working exactly?
Normally, you should just have to comment `WiFi.begin(WIFI_SSID, WIFI_PWD);` and `checkWifi()` in `main.cpp`, add instead `ETH.begin......` and maybe give few seconds to establish connection, add a delay(1000) after it. 
Maybe also just comment calls to checkWifi in the loop function as I'm not sure `WiFi.status()` would return WL_CONNECTED in case of Ethernet connection.

**Comment by bashers222** [comment]
Thanks, it boots but continues to connect to wifi.
I can see that code and the loops you mention in the man.cpp. I'll have a try later and see how i get on
Thanks for the quick response. Id love to have this connected by ethernet rather than wifi due to the very important role it plays in the home

**Comment by ap20132022** [comment]
interessting if you can put your code here after testing it. I have wifi and considering to change to eth because of stability.

**Reply by bashers222** [reply]
My first attempt at the code went into as boot loop, and I was not sure how to define the ETH pins as they all need assigning,
I'll be honest and say i'm not going to progress much down this path and instead have bought an ESP05 with socket for wifi antenna which will go outside the box.
